src/test/resources/company.txt

```diff
@@ -0,0 +1,7 @@
+Company[compName=My Company

```

Comment:
This file, and company.xml and company.xsd come in the IBM DFDL jar, so should be removed from here. Give instruction in the readme on how to setup.